libs/partners/chroma/langchain_chroma/vectorstores.py

```diff
@@ -941,7 +941,7 @@ def get(
         Args:
             ids: The ids of the embeddings to get. Optional.
             where: A Where type dict used to filter results by.
-                   E.g. `{"color" : "red", "price": 4.20}`. Optional.
+                   E.g. `{"$and": [{"color": "red"}, {"price": 4.20}]}` Optional.
```

Comment:
Does this not work?

```
`{"color" : "red", "price": 4.20}`.
```

It's supported by a lot of integrations to mean `$and`

Reply:
No this is not works with chromaDB. If you want to compare multiple argument then you compulsory need to use "$and".